### DISCUSSION TITLE
Is there a way to disable media sharing?

### DISCUSSION THREAD

**Discussion by mvdwetering** [timmo001/system-bridge#2880] [Q&A]
Hi, I just noticed that System bridge has an entry in the Home Assistant media browser.
I was a bit surprised by that. I just intended to monitor my system and there is no need for users in Home Assistant to browse media on that machine. 
So I would like to disable it.

I found the option in Settings to add more folders, but could not find a way to remove all folders or disable it completely.
Did I miss something or is it not possible yet?

This is on a Windows 10 machine btw

**Comment by Thunderhawk2001** [comment]
I agree with the question. I would also like to deactivate the sharing of my hard disks/media of my PC.

**Comment by Akio-42** [comment]
I would even go a step further and ask: is it possible to start system bridge in a read-only mode? No access to media and only getting monitoring information without the possibility to trigger any action on the monitored system?
Is this possible?

**Comment by MarkJaroski** [comment]
While we're at it, I would also like to disable reporting on loopback block devices like /sys etc. Is there a config file somewhere? Or would I need to run a personal fork and do this in python?
